packages/components/src/components/ScalarCombobox/ScalarComboboxOptions.vue

```diff
@@ -130,12 +138,8 @@ function moveActive(dir: 1 | -1) {
 }
 
 // Manual autofocus for the input
-const input = ref<HTMLInputElement | null>(null)
```

Comment:
@hwkr we needed the timeout here actually, its janky right now. bringing it back!